### ISSUE TITLE
slider menu item: enlarge fonts, icons, panel-height, etc. with one slider

### ISSUE THREAD

**Issue by sunweaver** [AyatanaIndicators/ayatana-indicator-a11y#14]
To ease readability of things on the screen, the a11y should have a slider for "resizing them all" (fonts, icons, panel site, etc.). The slider should be implemented for MATE only first, other DEs may follow.
